src/sentry/static/sentry/app/components/eventOrGroupHeader.jsx

```diff
@@ -109,9 +109,11 @@ class EventOrGroupHeader extends React.Component {
         style={data.status === 'resolved' ? {textDecoration: 'line-through'} : null}
       >
         {!hideLevel && level && (
-          <Tooltip title={`Error level: ${capitalize(level)}`}>
-            <GroupLevel level={data.level} />
-          </Tooltip>
+          <GroupLevel level={data.level}>
+            <Tooltip2 title={`Error level: ${capitalize(level)}`}>
+              <span />
```

Comment:
Because `GroupLevel` uses absolute positioning the tooltip anchor needs to be inside the positioned element or the tooltip will appear anchored around where the element would have been placed in document flow before positioning.